cypress/e2e/email.cy.ts

```diff
@@ -0,0 +1,23 @@
+describe('creation of a protected data with a valid email', () => {
+  it('as user i want to create a protected data', () => {
+    cy.contains('Login').click();
+    cy.viewport(1000, 660);
+    cy.get('@body').click(350, 600);
+    // as user i want to go to "My protected Data" and see the list of my protected data (represented by Cards)
+    cy.contains('Protect your data').click();
+    //as user i want to choose Email the data type i want with a submenu
+    cy.get('div[role="button"]').click();
+    //as user i want to choose "Email address" type
+    cy.contains('Email Address').click();
+    //as user i want to enter my mail address in a placeholder
+    cy.get('input[id="email"]').type('david.mialon@iex.ec');
+    //as user, i want to Name my protected data in a placeholder
+    cy.get('input[id="Name of your Protected Data"]').type('azerty');
```

Comment:
instead of putting azerty hardcoded you may want to add it to a variable : protectedDataNameMock